test.d

```diff
@@ -100,7 +100,7 @@ struct TestRunner
         version (Win64)
             auto result = executeShell("dub test --arch=x86_64");
         else
-            auto result = executeShell("dub test");
+            auto result = executeShell("dub test --arch=x86_mscoff");
```

Comment:
This will be run for all platforms except Windows 64bit, that's why the Travis CI tests are failing. `x86_mscoff` is only available on Windows, not on the Posix platforms.